gum/gumdarwinmodule.c

```diff
@@ -99,6 +120,114 @@ struct _GumExportsTrieForeachContext
   const guint8 * exports_end;
 };
 
+struct _GumChainedFixupsHeader
+{
+  uint32_t fixups_version;
+  uint32_t starts_offset;
+  uint32_t imports_offset;
+  uint32_t symbols_offset;
+  uint32_t imports_count;
+  GumChainedImportFormat imports_format;
+  GumChainedSymbolFormat symbols_format;
+};
+
+enum _GumChainedImportFormat
+{
+  GUM_CHAINED_IMPORT          = 1,
+  GUM_CHAINED_IMPORT_ADDEND   = 2,
+  GUM_CHAINED_IMPORT_ADDEND64 = 3,
+};
+
+struct _GumChainedImport
+{
+  uint32_t lib_ordinal :  8,
+           weak_import :  1,
+           name_offset : 23;
```

Comment:
This syntax may break MSVC compatibility, needs a closer look.

Reply:
aww i didn't think about windows at all. i may need some help / advice on this topic.